release.sh

```diff
@@ -206,6 +206,17 @@ docker run --rm \
 	gisaia/swagger-codegen-2.2.3 \
         -l python --type-mappings GeoJsonObject=object
 
+mkdir -p target/tmp/java-client
+
+docker run --rm \
+    -e GROUP_ID="$(id -g)" \
+    -e USER_ID="$(id -u)" \
+    --mount dst=/input/api.json,src="$PWD/target/tmp/swagger.json",type=bind,ro \
+    --mount dst=/input/config.json,src="$PWD/conf/swagger/java-config.json",type=bind,ro \
+    --mount dst=/output,src="$PWD/target/tmp/java-client",type=bind \
+	gisaia/swagger-codegen-2.3.1 \
+        -l java --type-mappings GeoJsonObject=Object
```

Comment:
Can we put the type-mappings into the java-config.json (in order to centralize the configuration)? I mean with something like `typeMappings: ["GeoJsonObject=Object"]`

Reply:
That would have been better but it is not supported in the list of properties that we can declare in java-config.json